pkgs/development/python-modules/billiard/default.nix

```diff
@@ -2,22 +2,19 @@
 
 buildPythonPackage rec {
   pname = "billiard";
-  version = "3.6.3.0";
+  version = "3.6.4.0";
   disabled = isPyPy;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0spssl3byzqsplra166d59jx8iqfxyzvcbx7vybkmwr5ck72a5yr";
+    sha256 = "0ismj2p8c66ykpss94rs0bfra5agxxmljz8r3gaq79r8valfb799";
   };
-  patches = [(fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/127419/commits/7d9db0318371adb4f9770f83802dc15b233b731e, will be squashed into https://github.com/NixOS/nixpkgs/pull/127419/commits/4c229018efb2d2c5f20f10f0bceefdf565d5448a before merge.